scripts/create-dotenv.sh

```diff
@@ -0,0 +1,60 @@
+DOTENV_PATH=./.env.xml
+
+UNITY_PATHS=($(find /Applications/Unity/hub -name "Unity.app"))
+
+if [ ${#UNITY_PATHS[@]} -eq '0' ] ; then 
+    echo "No Unity Editor was found"
+    exit 0
+fi
+
+echo "\nUnity Editor versions found"
+PS3='Choose a version : '
+select version in ${UNITY_PATHS[@]} quit; do
+    if [[ " ${UNITY_PATHS[*]} " =~ " ${version} " ]]; then
+        # whatever you want to do when array contains value
+        echo "$version selected"
+        break
+    fi
+    case $version in 
+        quit)
+            exit 0
+            ;;
+        *)
+            echo "Invalid option $REPLY"
+            ;;
+    esac
+done
+
+unity_dir=$(dirname "$version")
+unity_engine_dir="$version/Unity.app/Contents/Managed/UnityEngine/"
+dotenv_content="
+<Project>
+	<PropertyGroup>
+		<UNITY_DIR>$unity_dir/</UNITY_DIR>
+		<UNITY_ENGINE_DIR>$unity_engine_dir</UNITY_ENGINE_DIR>
+	</PropertyGroup>
+</Project>
+"
```

Comment:
Although not necessary, generally using [heredoc] is a more standard way of handling a multiline `string` in shell scripts.

```sh
dotenv_content=$(cat << EOF
<Project>
	<PropertyGroup>
		<UNITY_DIR>$unity_dir/</UNITY_DIR>
		<UNITY_ENGINE_DIR>$unity_engine_dir</UNITY_ENGINE_DIR>
	</PropertyGroup>
</Project>
EOF
)
```

[heredoc]: https://en.wikipedia.org/wiki/Here_document

Reply:
It's a very useful tool! I appreciate the comment on the multiline `string`